benchmarks/mmmu/mmmu.py

```diff
@@ -0,0 +1,169 @@
+import ast
+import base64
+from io import BytesIO
+
+from datasets import concatenate_datasets, load_dataset
+from PIL import Image
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample
+from inspect_ai.dataset._dataset import MemoryDataset
+from inspect_ai.model import ChatMessageUser, ContentImage, ContentText, GenerateConfig
+from inspect_ai.scorer import choice, model_graded_fact
+from inspect_ai.solver import generate, multiple_choice, prompt_template
+
+
+# Helper function to convert images to data URLs
+def image_to_data_url(image):
+    if isinstance(image, Image.Image):
+        buffered = BytesIO()
+        image.save(buffered, format="PNG")
+        image_bytes = buffered.getvalue()
+    else:
+        with open(image, "rb") as image_file:
+            image_bytes = image_file.read()
+
+    encoded_string = base64.b64encode(image_bytes).decode("utf-8")
+    mime_type = "image/png"
+    return f"data:{mime_type};base64,{encoded_string}"
+
+
+# Helper function to create content list from record
+def create_content_list(record):
+    content_list = [ContentText(text=record["question"])]
+    for i in range(1, 8):
```

Comment:
Why `range(1, 8)` here? Is there a maximum number of 8 images? Can't you just do `range(1, len(content_list))`?

Reply:
Yep, there's a maximum of seven images (seven image fields in the HF dataset).
`content_list` will only contain the question at this point in the code, unfortunately.